freqtrade/exchange/exchange.py

```diff
@@ -641,11 +670,21 @@ def get_min_pair_stake_amount(self, pair: str, price: float, stoploss: float,
         limits = market['limits']
         if ('cost' in limits and 'min' in limits['cost']
                 and limits['cost']['min'] is not None):
-            min_stake_amounts.append(limits['cost']['min'])
+            min_stake_amounts.append(
+                self._contracts_to_amount(
+                    pair,
+                    limits['cost']['min']
+                )
+            )
 
         if ('amount' in limits and 'min' in limits['amount']
                 and limits['amount']['min'] is not None):
-            min_stake_amounts.append(limits['amount']['min'] * price)
+            min_stake_amounts.append(
+                self._contracts_to_amount(
+                    pair,
+                    limits['amount']['min'] * price
+                )
+            )
```

Comment:
are we sure this (and the above) parts are needed?

can you point me to a case (exchange/pair combination) where we're getting cost/amount limits in contract size?

Reply:
So if you're familiar with the CCXT cli, just run `examples/js/cli okex market BTC/USDT:USDT` an you can see that the amount.min is `1` which is 1 BTC contract and not 1 BTC